include/roboteam_ai/control/positionControl/PositionControl.h

```diff
@@ -42,7 +42,7 @@ class PositionControl {
      * @param pidType The desired PID type (intercept, regular, keeper etc.)
      * @return a RobotCommand, which can be fed directly in the output
      */
-    RobotCommand computeAndTrackPath(const world::Field &field, int robotId, const Vector2 &currentPosition,
+    RobotCommand computeAndTrackPath(const rtt_world::Field &field, int robotId, const Vector2 &currentPosition,
             const Vector2 &currentVelocity, const Vector2 &targetPosition, stp::PIDType pidType);
```

Comment:
So there is an rtt_world and an rtt::world namespace? I think that can be confusing? Or is this a namespace alias?

Reply:
Namespace alias, did it cause otherwise i'd have to ::rtt::world evey time